pkg/signature/testdata/sign_file.sh

```diff
@@ -9,13 +9,22 @@ file=$1
 alwaysexcludes='.metadata.annotations."eksa.aws.com/signature"'
 has_excludes=$(yq "${file}" '.metadata.annotations."eksa.aws.com/excludes"')
 
+function debaser64 {
```

Comment:
If MacOS doesn't support `base64 -w0` we can simply use `base64 | tr -d '\n'` for both platforms instead since this will also work under Linux.

Reply:
Done